pkg/util/migrations/migrations.go

```diff
@@ -112,3 +111,17 @@ func VMIMigratableOnEviction(clusterConfig *virtconfig.ClusterConfig, vmi *v1.Vi
 	}
 	return false
 }
+
+func MigrationForVMI(migrationIndexer cache.Indexer, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstanceMigration, error) {
+	objs, err := migrationIndexer.ByIndex(cache.NamespaceIndex, vmi.Namespace)
+	if err != nil {
+		return nil, err
+	}
+	for _, obj := range objs {
+		migration := obj.(*v1.VirtualMachineInstanceMigration)
+		if migration.Spec.VMIName == vmi.Name {
+			return migration, nil
```

Comment:
Is this correct? What if we have multiple migration for the same VMI?

Reply:
There can't be multiple migrations for a VMI, the admitter won't let that happen.

Reply:
That's not correct. We could have multiple VMIMs created by the workload updater. Especially, when one fails, we  create new ones with an incremental time.